spec/compiler/codegen/case_spec.cr

```diff
@@ -13,6 +13,14 @@ describe "Code gen: case" do
     run("require \"prelude\"; case 1; when 0; 2; else; 3; end").to_i.should eq(3)
   end
 
+  it "codegens case without whens" do
+    run("require \"prelude\"; case 1; end").to_string.should eq ""
```

Comment:
I'm not sure it's a good idea to turn a `nil` into a string like this... it probably works, but it's not clear why (I wouldn't add a test for this if the normalizer is already tested, and same for the test below).

Reply:
`to_string` actually just returns `@output` of `Crystal::SpecRunOutput` which is already a string.

Reply:
But I can remove these specs, if their not considered particularly useful.